### DISCUSSION TITLE
Does Revanced supports android tv?

### DISCUSSION THREAD

**Discussion by sertchan** [ReVanced/revanced-discussions#528] [Question] [answered]
thats it

**Comment by TheJeterLP** [comment, marked as answer]
> thats it

No use smart tube next for that.

**Reply by sertchan** [reply]
https://github.com/yuliskov/SmartTubeNext this one right?

**Reply by TheJeterLP** [reply]
> https://github.com/yuliskov/SmartTubeNext this one right?

Yes exactly

**Reply by sertchan** [reply]
Im okay with this answer. You can close the discussion.
